yt/visualization/volume_rendering/lens.py

```diff
@@ -819,9 +821,67 @@ def set_viewpoint(self, camera):
         """
         self.viewpoint = camera.position
 
+
+class HEALPixLens(Lens):
+    def __init__(self):
+        super(HEALPixLens, self).__init__()
+        self.nside = 1
```

Comment:
For this healpix specific parameter `nside`, is it appropriate to set it here like this?

Reply:
If it's not going to be exposed to the user I'd just hardcode it in your call to `HEALPix` below. If you are going to expose it to the user then this makes sense.

Reply:
Yes, this will be exposed to the user to set the resolution.

Reply:
OK, in that case people will be able to customize it like this:

```
sc = yt.create_scene(ds, lens_type='HEALPix')
sc.camera.lens.nside = 8
```

You'll need to document that, of course.